pages/my-books/status/[classId].vue

```diff
@@ -88,31 +88,41 @@
           :rows="editionsTableRows"
         >
           <template #sort-data="{ row }">
-            <UButton
-              v-if="userIsOwner && prices.length > 1"
-              :icon="row.originalIndex === 0 ? 'i-heroicons-chevron-down' : 'i-heroicons-chevron-up'"
-              variant="ghost"
-              color="gray"
-              size="xs"
-              :label="String(row.originalIndex + 1)"
-              :disabled="isUpdatingPricesOrder || (row.originalIndex <= 0 && row.originalIndex >= prices.length - 1)"
-              :loading="isUpdatingPricesOrder"
-              trailing
-              @click="row.originalIndex === 0 ? movePriceDown(row.originalIndex) : movePriceUp(row.originalIndex)"
-            />
-            <span v-else v-text="String(row.originalIndex + 1)" />
+            <div v-if="row.originalIndex >= 0 && userIsOwner && prices.length > 1" class="flex flex-col gap-1">
```

Comment:
Instead of adding `row.originalIndex >= 0` everywhere, can we add a flag that says sth like `isStockBalancePlaceholderRow` to make this check more verbose?
Also please see if we just put one big `v-if` to handle this, instead of adding one for each `#sort-data`